masterlist.yaml

```diff
@@ -10820,6 +10820,33 @@ plugins:
         itm: 6
         udr: 164
 
+  - name: 'The Lost Spires.esp'
+    url:
+      - link: 'http://www.lostspires.com/pages/downloads.htm'
+        name: 'The Lost Spires'
+      - link: 'https://www.nexusmods.com/oblivion/mods/12997/'
+        name: 'The Lost Spires (Nexus Mods)'
+    after:
+      - 'Aundae.esp'
+      - 'Aundae_OBSE_addon.esp'
+      - 'Blood&Mud.esp'
+    msg:
+      - <<: *patchUpdateAvailable
+        subs: [ '[The Lost Spires - Cleaned Up Scribe Store Ruins](https://www.nexusmods.com/oblivion/mods/50146/)' ]
+        condition: 'not active("The Lost Spires - Cleaned Up Scribe.esp")'
+      - <<: *patchUpdateAvailable
+        subs: [ '[The Lost Spires - Enhanced Tomb](https://www.nexusmods.com/oblivion/mods/51414/)' ]
+        condition: 'not active("The Lost Spires - Enhanced Tomb.esp")'
```

Comment:
`patchUpdateAvailable` is more for official or unofficial updates or bug fixes for a mod than for add-ons or more subjective tweaks:

```suggestion
```